Makefile

```diff
@@ -53,7 +53,7 @@ check-licenses:
 .PHONY: check-licenses
 
 lint-python:
-	$(IN_CONDA_ENV) flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+	flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
You shouldn't need this change?

Reply:
Without that change, the command that runs < 1 sec on my Mac runs for more than 10 minutes with no output which times out Travis CI.

Reply:
@cclauss -- Something is going wrong there. It should run in almost the same time. How does `make README.rst` work for you?